osu.Game/Overlays/Profile/Header/Components/AddFriendButton.cs

```diff
@@ -49,10 +49,14 @@ private void load()
                     }
                 }
             };
+        }
 
-            User.BindValueChanged(user => updateFollowers(user.NewValue), true);
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            User.BindValueChanged(user => updateUser(user.NewValue), true);
         }
```

Comment:
why is this not a `UserBindingComponent`?

Reply:
This class already inherited from ProfileHeaderButton. I could just make a nested class for that, but wasn't sure. Also I followed your changes in https://github.com/ppy/osu/pull/5661 and you haven't made any changes related to this. So I wasn't sure twice